dialogue-core/src/main/java/com/palantir/dialogue/core/AimdConcurrencyLimiter.java

```diff
@@ -115,7 +122,10 @@ void dropped() {
          */
         void success() {
             inFlight.decrementAndGet();
-            limit.accumulateAndGet(inFlightSnapshot, LimitUpdater.SUCCESS);
+            int newLimit = limit.accumulateAndGet(inFlightSnapshot, LimitUpdater.SUCCESS);
+            if (log.isDebugEnabled()) {
+                log.debug("UP {}", SafeArg.of("newLimit", newLimit));
```

Comment:
Why do we have format characters in these log lines? I thought we had an errorprone rule to discourage you from doing so

Reply:
nit: better message, something like `limit increased.`

Reply:
My motivation for this PR is iterating on the SimulationTest to figure out better implementations for the ConcurrencyLimiters. We don't have sls-logging here in open-source land, so without these curly braces I'd just get:

INFO [main] com.palantir.dialogue.core.AimdConcurrencyLimiter - UP
INFO [main] com.palantir.dialogue.core.AimdConcurrencyLimiter - UP
INFO [main] com.palantir.dialogue.core.AimdConcurrencyLimiter - UP
INFO [main] com.palantir.dialogue.core.AimdConcurrencyLimiter - DOWN
INFO [main] com.palantir.dialogue.core.AimdConcurrencyLimiter - UP
INFO [main] com.palantir.dialogue.core.AimdConcurrencyLimiter - DOWN

Which doesn't give me enough useful signal.

Reply:
Ok sure, can we only add it to the debug lines then? Its kinda gross to have these unnecessary curlies in info logs we expect to be consumed through sls-logging.

Alternatively, we could open source sls-logging if feel like shaving a yak

Reply:
The majority of these log lines are debug already, so most people won't see them. The exception are things like `Exhausted {} retries, returning last received response with status {}`, which was already info.

I actually do want to keep the remaining curlies though this concretely helps me build stuff in this repo (and we've had explicit feedback that sls-logging style stuff in oss libraries is unhelpful https://github.com/palantir/dialogue/issues/700).